app-scientific/sagemath/autobuild/build

```diff
@@ -0,0 +1,8 @@
+export SAGE_LOCAL=/usr/lib/sagemath/local
+make configure
+./configure --disable-editable --prefix=$SAGE_LOCAL --enable-build-as-root
```

Comment:
Absolute path (`"$SRCDIR"/configure`) here, also please consider wrapping this long line with `\`:

```
"$SRCDIR"/configure \
    ARG1 \
    ARG2 \
    ARG3
```